git_pull/run.sh

```diff
@@ -217,8 +217,9 @@ cd /config || { echo "[Error] Failed to cd into /config"; exit 1; }
 while true; do
     check-ssh-key
     setup-user-password
-    git-synchronize
-    validate-config
+    git-synchronize; if [ $? -ne 1 ] ; then
```

Comment:
please use style: `if git-synchronize; then`

Reply:
you are totally right, that's much better. I changed it